### PR TITLE
fix: build test

### PR DESCRIPTION
**Pull request by ashnamehrotra**
updates trivy json file for build test

### CODE REVIEW

**Comment by sozercan** [comment]
@ashnamehrotra sounds like this failed because vulns are no longer at 0 since we are hardcoding the json, is that right? should this be generated instead of hardcoded?

**Comment by ashnamehrotra** [comment]
@sozercan yes thats right, it would be better to generate this file

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -30,6 +30,12 @@ jobs:
       - name: Pull docker.io/library/nginx:1.21.6
         run: docker pull docker.io/library/nginx:1.21.6
 
+      - name: Generate trivy vuln report for nginx image
+        run: |
+          cd ${{ github.workspace }}/test
+          mkdir data
+          trivy image --vuln-type os --ignore-unfixed -f json -o data/nginx.1.21.6.json docker.io/library/nginx:1.21.6
```
Comment (sozercan) [inline .github/workflows/build.yaml:37]:
```suggestion
          trivy image --vuln-type os --ignore-unfixed -f json -o /tmp/nginx.1.21.6.json docker.io/library/nginx:1.21.6
```

you can remove the 2 lines above

change references to `${{ github.workspace }}/test/data` below too

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -42,7 +46,7 @@ jobs:
           docker build --build-arg copa_version=${{ env.COPA_VERSION }} -t copa-action .
           cd ${{ github.workspace }}/test
```
Comment (sozercan) [inline .github/workflows/build.yaml:47]: we can remove this line too i think?

**Review by sozercan — APPROVED** [review]
one minor suggestion, otherwise LGTM. Thanks!

File: .github/workflows/build.yaml (hunk 2 above)
Comment (ashnamehrotra) [reply .github/workflows/build.yaml]: yup we can remove this, just need to change the bats command to call ./test/test.bats

**Pull request merged**